pkg/commands/workload_get.go

```diff
@@ -218,23 +219,40 @@ func (opts *WorkloadGetOptions) Exec(ctx context.Context, c *cli.Config) error {
 		}
 	}
 
-	pods := &corev1.PodList{}
-	err = c.List(ctx, pods, client.InNamespace(workload.Namespace), client.MatchingLabels{cartov1alpha1.WorkloadLabelName: workload.Name})
+	arg := []string{"Pod"}
+	r := c.Builder.Unstructured().
+		NamespaceParam(workload.Namespace).
+		LabelSelectorParam(fmt.Sprintf("%s%s%s", cartov1alpha1.WorkloadLabelName, "=", workload.Name)).
+		ResourceTypeOrNameArgs(true, arg...).
+		Latest().
+		Flatten().
+		TransformRequests(func(req *rest.Request) {
+			req.SetHeader("Accept", strings.Join([]string{
+				fmt.Sprintf("application/json;as=Table;v=%s;g=%s", metav1.SchemeGroupVersion.Version, metav1.GroupName),
+				"application/json",
+			}, ","))
+		}).
+		Do()
+	infos, err := r.Infos()
 	if err != nil {
 		c.Eprintf("\n")
 		c.Eerrorf("Failed to list pods:\n")
 		c.Eprintf("  %s\n", err)
 	} else {
-		if len(pods.Items) == 0 {
+		if len(infos) == 0 {
```

Comment:
This logic needs to be redone
- read the infos object and decide podList obj to be printed
- if podList lenght is empty then print `No pods...` msg 
else print the list of pods.
- Looks like you are expecting infos to have 1 podList object in this case so explicitly call `				tableResult, podlist, _ := printer.DecodeIntoTable(infos[i0].Object)
`

Reply:
Done